src/components/composer/content_editable_helper.ts

```diff
@@ -190,7 +194,7 @@ export class ContentEditableHelper {
     const focusedNode = document.getSelection()?.focusNode;
     if (!focusedNode || !this.el.contains(focusedNode)) return;
     const element = focusedNode instanceof HTMLElement ? focusedNode : focusedNode.parentElement;
-    element?.scrollIntoView({ block: "nearest" });
+    element?.scrollIntoView?.({ block: "nearest" });
```

Comment:
jsdom?

Reply:
yes, it's not implemented (`element.scrollIntoView` is undefined)